Updated ARC related profiles (#369)
ResearchObject/ro-crate: docs/pages/resources/profiles.md

## ARC RO-Crate profile

The [ARC RO-Crate profile](https://github.com/nfdi4plants/arc-ro-crate-profile) describes [Annotated Research Contexts](https://arc-rdm.org/) (ARC), developed by [DataPLANT](https://nfdi4plants.org/).
An ARC consists of [ISA](https://isa-specs.readthedocs.io/en/latest/isamodel.html) metadata describing the experimental setup and computational workflows given in CWL.
The current profile is a combination of the [ISA RO-Crate profile](https://github.com/nfdi4plants/isa-ro-crate-profile) and the [ARC CWL RO-Crate profile](https://github.com/nfdi4plants/arc-cwl-ro-crate-profile), see details below.
It requires the crate to follow the ISA Investigation profile on the top level.
The investigation object contains studies and assays from the ISA model, as well as computational workflows.

> The core entity of both sub-profiles is the type `LabProcess` defined by [bioschmeas](https://bioschemas.org/) and used to describe processes leading to data files in a parametrized way.

Reviewer: ```suggestion
The core entity of both sub-profiles is the type `LabProcess` defined by [Bioschemas](https://bioschemas.org/) and is used to describe processes leading to data files in a parametrized way.
```

Author: To me, the second 'is' seems wrong here. Maybe change it to "The core entity of both sub-profiles is the type `LabProcess` defined by Bioschemas, which is used to describe processes leading to data files in a parametrized way"?

Reviewer: sure, updated!

How such an RO-Crate can be generated from an ARC is described in the [arc-to-rocrate](https://github.com/nfdi4plants/arc-to-rocrate) repository, which also contains scripts to perform the conversion.

## ISA RO-Crate Profile

The [ISA RO-Crate profile](https://github.com/nfdi4plants/arc-to-rocrate/blob/main/profiles/investigation.md) describes experimental data in plant sciences that is annotated by metadata following the [ISA model](https://isa-specs.readthedocs.io/en/latest/isamodel.html).
Such datasets consist of three types of data entities: `Investigation`, `Study` and `Assay`.
The profile adds requirements of the crate such that the data folders match the Investigation, Study and Assay objects of the [ISA model](https://isa-specs.readthedocs.io/en/latest/isamodel.html).

> The description of experimental processes that produce the data uses the [bioschmeas](https://bioschemas.org/) type `LabProcess` as its core entity.

Reviewer: ```suggestion
The description of experimental processes that produce the data uses the [Bioschemas](https://bioschemas.org/) type `LabProcess` as its core entity.
```

It describes how data and samples are derived from each other in a graph model based on formalized parameters as key-value pairs.
Profiles for all included types can be found in the [full version](https://github.com/nfdi4plants/isa-ro-crate-profile).

## ARC CWL RO-Crate Profile

> A [ARC CWL RO-Crate profile](https://github.com/nfdi4plants/arc-cwl-ro-crate-profile) extends the existing Worflow Run Crate in an ARC-specific context.

Reviewer: ```suggestion
The [ARC CWL RO-Crate profile](https://github.com/nfdi4plants/arc-cwl-ro-crate-profile) extends the existing [Workflow Run Crate profile](https://w3id.org/ro/wfrun/workflow/) in an ARC-specific context.
```

> It uses the same process graph types (`LabProtocol`,`LabProcess`) as the ISA profile to further annotate parameters of workflows (equivalent to protocols in ISA) and runs (equivalent to processes in ISA).  

Reviewer: ```suggestion
It uses the same process graph types (`LabProtocol`,`LabProcess`) as the [ISA profile](#isa-ro-crate-profile) to further annotate parameters of workflows (equivalent to protocols in ISA) and runs (equivalent to processes in ISA).  
```
